.eslintrc

```diff
@@ -37,6 +37,7 @@
       "requireReturnDescription": true
     }],
   "linebreak-style": "off",
+  "import/no-extraneous-dependencies": "off",
```

Comment:
I'm not sure what you mean by this, using the word extraneous seems a little over-kill and doesn't help add definition to this line.